### PR TITLE
Opening Days Module

### PR DESCRIPTION
**Pull request by MarianaLopez-Ksquare**

### CODE REVIEW

**Review by ArgenisGonzalez-Ksquare — DISMISSED** [review]

**Review by YumilRuedaFlores-Ksquare — DISMISSED** [review]

**Review by ArgenisGonzalez-Ksquare — APPROVED** [review]

File: src/models/OpeningDays.model.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            primaryKey: true
+        },
+        restaurant_id :{
+            type:DataTypes.INTEGER,
+            autoIncrement: true,
+            primaryKey: true
+        }
+
+}, {
+    sequelize // Instance of sequelize that reflects the connection
```
Comment (GabrielOsorno-Ksquare) [inline src/models/OpeningDays.model.ts:25]: I think we should all add tableName, so all the table names are standardized at snake_case

File: src/models/OpeningDays.model.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Model, InferAttributes, InferCreationAttributes, CreationOptional, DataTypes, Sequelize } from "sequelize";
+
+
+export class OpeningDays extends Model<InferAttributes<OpeningDays>, InferCreationAttributes<OpeningDays>> {
+
+    declare id: number;
```
Comment (GabrielOsorno-Ksquare) [inline src/models/OpeningDays.model.ts:6]: Maybe CreationOptional?

File: src/models/OpeningDays.model.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        id: {
+            type: DataTypes.INTEGER,
+            autoIncrement: true,
+            primaryKey: true
+        },
+        restaurant_id :{
+            type:DataTypes.INTEGER,
+            autoIncrement: true,
+            primaryKey: true
+        }
```
Comment (GabrielOsorno-Ksquare) [inline src/models/OpeningDays.model.ts:22]: Two primary keys?

**Review by GabrielOsorno-Ksquare — CHANGES_REQUESTED** [review]

**Review by ArgenisGonzalez-Ksquare — APPROVED** [review]

File: src/models/OpeningDays.model.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+        id: {
+            type: DataTypes.INTEGER,
+            autoIncrement: true,
+            primaryKey: true
+        },
+        restaurant_id :{
+            type:DataTypes.INTEGER,
+            allowNull: false,
+        },
```
Comment (YumilRuedaFlores-Ksquare) [inline src/models/OpeningDays.model.ts:22]: Add is_deleted

**Review by GabrielOsorno-Ksquare — APPROVED** [review]

**Review by YumilRuedaFlores-Ksquare — APPROVED** [review]

**Review by GenaroArredondo-Ksquare — APPROVED** [review]

**Review by GabrielOsorno-Ksquare — APPROVED** [review]

**Pull request merged**